In [1]:
# import libraries
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
import missingno as msno

In [2]:
# function to clean column names
def clear_cols(s):
    return s.strip().replace(" ","_")

# load data
path = '../Healthcare Provider Fraud Detection Analysis/Test-1542969243754.csv'
df = pd.read_csv(path).rename(columns=clear_cols)


In [3]:
print(f"Shape of the file: {df.shape}")
df.head()

Shape of the file: (1353, 1)


Provider
0  PRV51002
1  PRV51006
2  PRV51009
3  PRV51010
4  PRV51018

In [4]:
print(f"duplicates -> {df.duplicated().sum()}\n")
print(f"null values: \n{df.isnull().sum()}")

duplicates -> 0

null values: 
Provider    0
dtype: int64


In [5]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1353 entries, 0 to 1352
Data columns (total 1 columns):
 #   Column    Non-Null Count  Dtype 
---  ------    --------------  ----- 
 0   Provider  1353 non-null   object
dtypes: object(1)
memory usage: 10.7+ KB


In [6]:
# df['PotentialFraud'].value_counts()

In [7]:
# presentage of missing values
df.isna().sum()/len(df) *100

Provider    0.0
dtype: float64

In [8]:
df['Provider'].nunique()

1353

* This file contain one column of Provider. It's purpose is to use only to the testing part and will be merged only when test the models.
* Contain 1,353 providers from the other datasets but comes without the indication of `Fraud` or `Not Fraud`
